.drone.star

```diff
@@ -2884,6 +2884,14 @@ def k6LoadTests(ctx):
                     "sh %s/run_k6_tests.sh --ocis-log" % (dirs["base"]),
                 ],
             },
+            {
+                "name": "open-grafana-dashboard",
+                "image": OC_CI_ALPINE,
+                "environment": {},
+                "commands": [
+                    "echo `Grafana Dashboard: <https://grafana.k6.infra.owncloud.works>`_",
```

Comment:
Hopefully I'll see it as a link (not as text)

https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-custom-link-text)https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#use-custom-link-text

Reply:
I doubt if it works. I got this error in my term
```bash
$ echo `Grafana Dashboard: <https://grafana.k6.infra.owncloud.works>`_
zsh: parse error near `>'
zsh: parse error in command substitution
```

Reply:
left as text